include/follow.php

```diff
@@ -82,6 +82,11 @@ function new_contact($uid,$url,$interactive = false) {
 		return $result;
 	}
 
+	if (blocked_url($url)) {
+		$result['message'] = t('Blocked domain');
```

Comment:
I thought about it, however all the other messages are very succinct to the point they're unhelpful, and I didn't want to improve this part of the system or I would go on an edit rampage. The other reason I came up with is that the reason is in a single language, it would be pretty ugly if the translated label was followed by the untranslated reason. The final reason is that the block reason is useless for users, they can't act on it anyway. It would make more sense to show the reason to the requesting server, but this is out of the scope of this PR and unlikely to be helpful either.